cypress/locators/menu/locators.js

```diff
@@ -2,3 +2,4 @@
 export const MENU_PREVIEW = '[data-component="menu"]';
 export const MENU_ITEM = '[data-component="menu-item"]';
 export const SUBMENU = '[data-component="submenu-wrapper"]';
+export const SCROLL_BLOCK = 'div[overflow="auto"]';
```

Comment:
are you sure this is a correct locator ?

Reply:
Yep, it finds the right element. There is probably a more unique way to locate it though